src/main/java/com/zombiecastlerush/building/Room.java

```diff
@@ -7,6 +7,7 @@
 public class Room {
     private String name;
     private String description;
+    Inventory inventory;
```

Comment:
Does this need to be public?

Reply:
I'm mainly said this, because I forgot, lol. I can't remember if we have to do this or not. Since we are planning to do a player.inventory.getItems(), and a room.inventory.getItems(), I just want to make sure those methods can actually work.

Reply:
I actually made it public, it's all good.

Reply:
thanks